ui/components-react/src/test/filter-builder/FilterBuilderRuleGroup.test.tsx

```diff
@@ -34,31 +40,39 @@ describe("PropertyFilterBuilderRuleGroupRenderer", () => {
   });
 
   it("does not render remove button for root group", () => {
-    const {queryByTestId} = renderWithContext(<PropertyFilterBuilderRuleGroupRenderer {...defaultProps} />);
+    const { queryByTestId } = renderWithContext(<PropertyFilterBuilderRuleGroupRenderer {...defaultProps} />);
 
     expect(queryByTestId("rule-group-remove")).to.be.null;
   });
 
   it("renders remove button for non root group", () => {
-    const {getByTestId} = renderWithContext(<PropertyFilterBuilderRuleGroupRenderer
+    const { getByTestId } = renderWithContext(<PropertyFilterBuilderRuleGroupRenderer
       {...defaultProps}
-      group={{id: "id", groupId: "parentId", items: [], operator: PropertyFilterRuleGroupOperator.And}}
+      group={{ id: "id", groupId: "parentId", items: [], operator: PropertyFilterRuleGroupOperator.And }}
     />);
     getByTestId("rule-group-remove");
   });
 
   it("does not render add group button if depth limit is reached", () => {
-    const {queryByTestId} = renderWithContext(<PropertyFilterBuilderRuleGroupRenderer
+    const { queryByTestId } = renderWithContext(<PropertyFilterBuilderRuleGroupRenderer
       {...defaultProps}
       path={["parentId", "id"]}
-      group={{id: "id", groupId: "parentId", items: [], operator: PropertyFilterRuleGroupOperator.And}}
-    />, {ruleGroupDepthLimit: 1});
+      group={{ id: "id", groupId: "parentId", items: [], operator: PropertyFilterRuleGroupOperator.And }}
+    />, { ruleGroupDepthLimit: 1 });
     expect(queryByTestId("rule-group-add-rule-group")).to.be.null;
   });
 
+  it("does not render operator selector if only one rule is in group", () => {
+    const { queryByText } = renderWithContext(<PropertyFilterBuilderRuleGroupRenderer
+      {...defaultProps}
+      group={{ id: "id", items: [{ id: "childId", groupId: "id" }], operator: PropertyFilterRuleGroupOperator.And }}
+    />);
+    expect(queryByText(TestUtils.i18n.getLocalizedString("Components:filterBuilder.operators.or"))).to.be.null;
+  });
+
```

Comment:
I dont see a test that validates that this would ever occur, even if there was more that 1 item in the group at this level.

(If there was multiple items in the group, I'm pretty sure this test would still succeed as we are look for the "or" string rather than the "and", that may be expected, but given that there is no comparison, it look like a simple error.)

Reply:
Thanks, nice catch. It should look for "and" operator. Fixed it.